maximize-select2-height.min.js

```diff
@@ -1,8 +1,9 @@
-// maximize-select2-height v1.0.3
-// (c) Panorama Education 2018
+// maximize-select2-height v1.0.4
+// (c) Panorama Education 2020
 // MIT License
 !function(t){"use strict"
-var e=t(window),n=t(document),o=function(o,i){return t.extend({cushion:i&&n.width()>e.width()?30:10},o)},i=function(n,i,c,r,s){var u,h,l
-return s?u=window.document.documentElement.clientHeight+e.scrollTop()-i.offset().top:(l=t("#select2-"+n+"-container").parent().parent().parent().offset().top,h=c.height()-i.height(),u=l-e.scrollTop()-h),u-o(r,s).cushion}
-t.fn.maximizeSelect2Height=function(e){return this.each(function(n,o){t(o).on("select2:open",function(){setTimeout(function(){var n=t("#select2-"+o.id+"-results"),c=n.parent(),r=c.parent(),s=r.hasClass("select2-dropdown--below"),u=i(o.id,n,r,e,s)
-c.css("max-height",u),n.css("max-height",u),t(document).trigger("scroll")})})})}}(jQuery)
+var e=t(window),n=t(document),o=function(o,i){return t.extend({cushion:i&&n.width()>e.width()?30:10},o)},i=function(t,n,i,c,s){var r,u,h
```

Comment:
Produced this file by pasting the previous file into `UglifyJS2` as usual here: http://lisperator.net/uglifyjs/